doc/source/drivers/vector/sqlite.rst

```diff
@@ -227,6 +227,14 @@ Transaction support
 
 The driver implements transactions at the database level, per :ref:`rfc-54`
 
+Relationships
+-------------
+
+.. versionadded:: 3.6
+
+Relationship retrieval is supported. Relationships will be reported for tables which utilise FOREIGN KEY
+constraints, and have ``PRAGMA foreign_keys = ON``.
```

Comment:
I don't understand the PRAGMA foreign_keys = ON requirements. This is a volatile/non-serialized pragma.

cf
```
$ ~/install-sqlite-3.39.0/bin/sqlite3 foo.db
SQLite version 3.39.0 2022-06-25 14:57:57
Enter ".help" for usage hints.
sqlite> pragma foreign_keys;
0
sqlite> pragma foreign_keys = on;
sqlite> pragma foreign_keys;
1

$ ~/install-sqlite-3.39.0/bin/sqlite3 foo.db
SQLite version 3.39.0 2022-06-25 14:57:57
Enter ".help" for usage hints.
sqlite> pragma foreign_keys;
0
``